src/firefly/html/demo/ffapi-highlevel-test.html

```diff
@@ -23,6 +23,14 @@
     <br>
 </div>
 
+<div>
+    <!--<a href='javascript:firefly.getViewer("testChannel").showImage({url:"http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band2.fits"})'>test open</a>-->
+    <a href='javascript:firefly.getViewer().plotURL("http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band2.fits")'>test open</a>
```

Comment:
>> opens new app with the image in expanded view but the search drop-down on top of it

I made a ticket, DM-6108.  There are several more issues the need to be worked in controlling the tri-view layout.